package.json

```diff
@@ -54,13 +54,15 @@
     "eslint-plugin-react": "^6.10.3",
     "jest": "^23.4.0",
     "prettier": "^1.2.2",
+    "prop-types": "^15.0.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0"
   },
   "dependencies": {
     "focus-trap": "^3.0.0"
   },
   "peerDependencies": {
+    "prop-types": "^15.0.0",
```

Comment:
It this absolutely necessary, or could we avoid the breaking change by providing a dummy function for the context types?

Maybe we should end up adding a PropTypes peer dependency, but I'd love to treat that as a separate issue if possible.